test/unit/webgl/p5.Camera.js

```diff
@@ -0,0 +1,540 @@
+suite('p5.Camera', function() {
+  var myp5;
+  var myCam;
+  var delta = 0.001;
+
+  // copy of p5.Vector.prototype.angleBetween...
+  var angleBetween = function(a1, a2, a3, b1, b2, b3) {
```

Comment:
hmm, I think it ought to be ok to just use the vector version (and make some vectors) when you need this? unless I'm missing something?

Reply:
Ok.  I was trying to keep it isolated, but not for any particular reason!